app/components/Request/messages.js

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from 'react-intl'
+
+/* eslint-disable max-len */
+export default defineMessages({
+  amount: 'Amount',
+  button_text: 'Request',
+  copy_Address: 'Copy Address',
+  copied_title: 'Address copied',
+  copied_description: 'Payment address has been copied to your clipboard',
+  ln_address: 'Lightning Request Address',
+  total: 'Total',
+  memo: 'Memo',
+  memo_placeholder: 'For example "Dinner last night"',
+  not_paid: 'not paid',
+  paid: 'paid',
+  qrcode: 'QR-Code',
+  request_status: 'Request Status',
+  title: 'Request',
+  subtitle: 'through Lightning Network',
+  description:
+    'You can request Bitcoin (BTC) through the Lightning Network. Just enter the Amount you want to request in the field below. Zap will generate a QR-Code and a Lightning Request Address after.'
```

Comment:
Instead of `Lightning Request Address` how about will generate a `Lightning invoice`...